File: mobile/app/src/main/java/com/edvaldotsi/fastfood/model/Contato.java
package com.edvaldotsi.fastfood.model;

import java.io.Serializable;

/**
 * Created by Edvaldo on 22/09/2015.
 */
public class Contato implements Serializable {

    private int codigo;
    private String telefone;
    private transient Cliente cliente;

    public Contato() {}

    public Contato(String telefone) {
        this.telefone = telefone;
    }

    public int getCodigo() {
        return codigo;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

    public String getTelefone() {
        return telefone;
    }

    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }

    public Cliente getCliente() {
        return cliente;
    }

    public void setCliente(Cliente cliente) {
        this.cliente = cliente;
    }

    @Override
    public String toString() {
        return telefone;
    }
}

File: fast_food/module/Application/Model/Contato.php
<?php

namespace Application\Model;

class Contato
{
    /**
     * @var int
     */
    private $codigo;

    /**
     * @var string
     */
    private $telefone;

    /**
     * @var Cliente
     */
    private $cliente;

    /**
     * Contato constructor.
     * @param int $codigo
     */
    public function __construct($codigo)
    {
        $this->codigo = $codigo;
    }

    /**
     * @return int
     */
    public function getCodigo()
    {
        return $this->codigo;
    }

    /**
     * @param int $codigo
     */
    public function setCodigo($codigo)
    {
        $this->codigo = $codigo;
    }

    /**
     * @return string
     */
    public function getTelefone()
    {
        return $this->telefone;
    }

    /**
     * @param string $telefone
     */
    public function setTelefone($telefone)
    {
        $this->telefone = $telefone;
    }

    /**
     * @return Cliente
     */
    public function getCliente()
    {
        return $this->cliente;
    }

    /**
     * @param Cliente $cliente
     */
    public function setCliente($cliente)
    {
        $this->cliente = $cliente;
    }
}